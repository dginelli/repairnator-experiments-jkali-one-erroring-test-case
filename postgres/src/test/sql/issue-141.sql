SELECT *
FROM zdb_highlight('so_posts', 'body:("coworker" w/4 ("that" OR "this"))', 'id=1', '{body}')
ORDER BY "primaryKey", "fieldName", "arrayIndex", "position";