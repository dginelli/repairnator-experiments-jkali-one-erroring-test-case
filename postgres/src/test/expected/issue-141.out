SELECT *
FROM zdb_highlight('so_posts', 'body:("coworker" w/4 ("that" OR "this"))', 'id=1', '{body}')
ORDER BY "primaryKey", "fieldName", "arrayIndex", "position";
 primaryKey | fieldName | arrayIndex |   term   |    type    | position | startOffset | endOffset |        clause        
------------+-----------+------------+----------+------------+----------+-------------+-----------+----------------------
 1          | body      |          0 | coworker | <ALPHANUM> |        3 |           5 |        13 | body CONTAINS "null"
 1          | body      |          0 | that     | <ALPHANUM> |        7 |          31 |        35 | body CONTAINS "null"
(2 rows)

